mcgj/mcgj.py

```diff
@@ -28,7 +28,15 @@ def index():
 @bp.route("/profile")
 @login_required
 def profile():
-    return render_template("edit_profile.html", user=current_user)
+    user_tracks_query = "SELECT * FROM tracks WHERE user_id = ? AND cue_date IS NOT NULL ORDER BY cue_date DESC LIMIT 50"
```

Comment:
Eventually it would be really cool to do pagination to let folks browse through their full history, but passing in the last 50 seems very reasonable!

Reply:
Yeah, this was the real quick and dirty to get something useful, but I'm happy to add pagination in a later PR this week!